src/portpicker.py

```diff
@@ -224,7 +224,11 @@ def get_port_from_port_server(portserver_address, pid=None):
 
     try:
         # Create socket.
-        sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        if getattr(socket, 'AF_UNIX', False):
```

Comment:
if you `getattr(socket, 'AF_UNIX', False)` and `socket.AF_UNIX` exists and is `0`, the if on the getattr result will jump to the `else:` in the same manner as if `socket.AF_UNIX` did not exist.

I'm assuming the actual purpose of this line of code is to test if AF_UNIX exists or not rather than what value it has.

Reply:
Ah, thanks for the explanation. I'll update the PR.

Reply:
PR updated.